alert-common/src/main/java/com/synopsys/integration/alert/common/channel/message/ChannelMessageParser.java

```diff
@@ -77,14 +79,24 @@ public String createHeader(MessageContentGroup messageContentGroup) {
         return headerSeparator;
     }
 
-    public String getCommonTopic(MessageContentGroup messageContentGroup) {
-        return createLinkableItemString(messageContentGroup.getCommonTopic(), true) + getLineSeparator();
+    public String getCommonTopic(MessageContentGroup messageContent, ItemOperation nullableTopLevelAction) {
+        LinkableItem commonTopic = messageContent.getCommonTopic();
+        if (ItemOperation.DELETE.equals(nullableTopLevelAction)) {
+            commonTopic = new LinkableItem(commonTopic.getName(), commonTopic.getValue());
+        }
```

Comment:
When should we use the ternary operator vs something like this?

Reply:
This is gonna sound weird, but I think a "three dot" maximum should be the rule of thumb. If dereferencing is happening more than three times, it should not be part of a ternary operation.

Reply:
That goes for the calling class as well, think of method calls as `this.doSomething()`.